src/App.js

```diff
@@ -10,9 +14,21 @@ class CalcApp extends React.Component {
 
   render() {
     return (
-      <div className="ourApp">
-        <OurCalcApp />
-      </div>
+      <>
+        <Router>
+          <Navigation />
+          <section className="content">
+            <Routes>
+              <Route path="/math-magician" element={<Home />} />
```

Comment:
- [ ] **[OPTIONAL]** Your app displays nothing when I open it. I think it would be better to change this route to the root route like (remember to change the right link in the navbar as well):

  ```
  <Route path="/" element={<Home />} />
  ```

  ![rro](https://user-images.githubusercontent.com/81468675/191524313-a7f3997d-8710-4987-8569-8ad09ee1e0c7.jpg)